caput/time.py

```diff
@@ -228,6 +228,11 @@ def __init__(self, lon=0.0, lat=0.0, alt=0.0, lsd_start=None, sf_wrapper=None):
 
     _obs = None
 
+    @property
```

Comment:
The implementation looks good, but I'm not sure if this should be a property for reasons related to the comments in the other PR.

Reply:
Given that there isn't any specific reason why it needs to be a property, I'm going to remove that